.github/workflows/push.yml

```diff
@@ -77,16 +77,19 @@ jobs:
         name: Tests
         run: docker run --rm ${{ env.APP_IMAGE}} composer ci
 
-  tests-in-kbc:
-    needs: build
+  deploy:
+    needs:
```

Comment:
Chcel som tu povodne vymazat `tests-in-kbc`, ale vymazal som `deploy` :see_no_evil:. Fixnute.
Popri ZD by som asi nemal posielat PR.